examples/ad9371-iiostream.c

```diff
@@ -85,7 +85,7 @@ static void shutdown()
 
 static void handle_sig(int sig)
 {
-	printf("Waiting for process to finish...\n");
+	printf("Waiting for process to finish... Got signal %d\n",sig);
```

Comment:
nitpick:  printf("Waiting for process to finish... Got signal %d\n",`__space__`sig);

Reply:
fixed